### PR TITLE
feat(predictions): Card state for house wins

### PR DESCRIPTION
**Pull request by hachiojidev**
image

### CODE REVIEW

File: src/views/Predictions/helpers.ts [hunk 1]
```diff
@@ -84,3 +84,15 @@ export const getPriceDifference = (price: ethers.BigNumber, lockPrice: ethers.Bi
...
+export const getRoundPosition = (lockPrice: ethers.BigNumber, closePrice: ethers.BigNumber) => {
+  if (!closePrice) {
+    return null
+  }
+
+  if (closePrice.eq(lockPrice)) {
+    return BetPosition.HOUSE
+  }
+
+  return closePrice.gt(lockPrice) ? BetPosition.HOUSE : BetPosition.HOUSE
```
Comment (ChefKai) [inline src/views/Predictions/helpers.ts:97]: ?

Comment (hachiojidev) [reply src/views/Predictions/helpers.ts:97]: Nice catch. Was using that for testing!

**Review by RabbitDoge — APPROVED** [review]

File: src/views/Predictions/components/RoundResult/styles.tsx [hunk 2]
```diff
@@ -140,6 +143,41 @@ export const RoundResultBox: React.FC<RoundResultBoxProps> = ({
...
+export const RoundPrice: React.FC<RoundPriceProps> = ({ lockPrice, closePrice }) => {
+  const betPosition = getRoundPosition(lockPrice, closePrice)
+  const priceDifference = getPriceDifference(closePrice, lockPrice)
+
+  const getTextColor = () => {
+    switch (betPosition) {
+      case BetPosition.BULL:
+        return 'success'
+      case BetPosition.BEAR:
+        return 'success'
```
Comment (Chef-Cheems) [inline src/views/Predictions/components/RoundResult/styles.tsx:160]: I think this one should be `failure`

**Review by Chef-Cheems — APPROVED** [review]

File: src/views/Predictions/components/RoundResult/styles.tsx (hunk 2 above)
Comment (hachiojidev) [reply src/views/Predictions/components/RoundResult/styles.tsx]: Another great catch.

**Pull request merged**